Library/UpdateBackingInput+Constructor.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import KsApi
+
+extension UpdateBackingInput {
+  internal static func input(
+    from updateBackingData: UpdateBackingData
+  ) -> UpdateBackingInput {
+    let pledgeAmount: String? = updateBackingData.pledgeAmount.flatMap { pledgeAmount in
+      pledgeAmountString(withAmount: pledgeAmount, shippingRule: updateBackingData.shippingRule)
+    }
+    let backingId: String = updateBackingData.backing.graphID
+    let locationId: String? = updateBackingData.shippingRule.flatMap { "\($0.location.id)" }
+    let rewardId: String? = updateBackingData.reward.graphID
+
+    return UpdateBackingInput(
+      amount: pledgeAmount,
+      applePay: nil,
+      id: backingId,
+      locationId: locationId,
+      paymentSourceId: nil,
+      rewardId: rewardId
+    )
+  }
+}
+
+func pledgeAmountString(withAmount amount: Double, shippingRule: ShippingRule?) -> String {
```

Comment:
Should we either make this `private` and/or `test` it? 🤔

Reply:
Going to make it private for now and consider it tested by way of this constructor being tested.

This will become a shared function when we add the newer `CreateBacking` mutation and then I'll break it out and write tests for it specifically.